PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackCuts.cxx

```diff
@@ -1009,7 +1018,7 @@ AliFemtoDreamTrackCuts* AliFemtoDreamTrackCuts::PrimProtonCuts(
   trackCuts->SetDCAVtxXY(0.1);
   trackCuts->SetCutSharedCls(true);
   trackCuts->SetCutTPCCrossedRows(true, 70, 0.83);
-  trackCuts->SetPID(AliPID::kProton, 0.75);
+  trackCuts->SetPID(AliPID::kProton, 0.75, 3.);
```

Comment:
why do you change this? If the default values of this function were modified many things will no longer work....